README.md

```diff
@@ -1,39 +1,106 @@
+
+| Inference Type | Client | Parallelization |
+| :--- | --- | --- |
+| Standard | C++ AND Fortran | MPI |
+| Colocated | C++ AND Fortran | MPI |
+
+#### `Throughput Tests`
+
+| Throughput Type | Client | Parallelization |
+| :--- | --- | --- |
+| Standard | C++ | MPI |
+| Colocated | C++ | MPI |
+
+#### `Data Aggregation Tests`
+
+| Data Aggregation Type | Client | Parallelization |
+| :--- | --- | --- |
+| Standard | C++ | MPI |
+| Standard | Python | MPI |
+| Standard | Python | File System |
 
+## Colocated vs Standard Deployement
 
-## Scaling Tests
+The scaling repo offers two types of Orchestrator deployment: Standard and Colocated.
 
-There are two types of scaling tests in the repository.
+> Note that the Orchestrator is a SmartSim term for a Redis or KeyDB database with a RedisAI module built into it with the ML runtimes.
 
- 1. Inference
- 2. Throughput
+1. `Colocated (non-Clustered Deployement)`
+  : A colocated Orchestrator is a special type of Orchestrator that is deployed on the same compute hosts as the application. 
+  This is particularly important for GPU-intensive workloads which require frequent communication with the database. You can specify the number of nodes via the `client_nodes` flag. 
 
-Both applications use a MPI + C++ application to mimic an HPC workload
-making calls to SmartSim infrastructure. These applications are used
-to test the performance of SmartSim across various system types.
+2. `Standard (Clustered Deployement)`
+  : When running with Standard deployment, your database will be deployed on different compute nodes
+than your application. You will notice that all Standard scaling tests share a `db_nodes` flag. By setting the flag to `db_nodes=[4,8]` - you are telling the program to split up your database to four shards on the first permutation, then eight shards on the second permutation. Each shard of the database will communicate with each application node. You can specify the number of application nodes via the `client_nodes` flag in each scaling test.
+
+See [our installation docs](https://www.craylabs.org/docs/orchestrator.html) for 
+more information on clustered and colocated deployment
 
 ## Building
 
-To run the scaling tests, SmartSim and SmartRedis will need to be
-installed. See [our installation docs](https://www.craylabs.org/docs/installation.html)
-for instructions.
+**To run the scaling tests, SmartSim and SmartRedis will need to be
+installed.** See [our installation docs](https://www.craylabs.org/docs/installation_instructions/basic.html) for instructions.
 
 For the inference tests, be sure to have installed SmartSim with support
 for the device (CPU or GPU) you wish to run the tests on, as well as
-have built support for the PyTorch backend.
+have built support for the PyTorch backend. If you would also like to 
+run the Fortran tests, make sure to run `make lib-with-fortran` from 
+the install docs for SmartRedis. For C++ scaling tests, running
+`make lib` is sufficient.
```

Comment:
Move the additional SmartRedis instructions to the bit about installing SmartSim and SmartRedis. They feel disjoint here

Reply:
so I am not sure if it would make 100% sense up there, I moved it below, let me know what u think, its a comment in the install example, actually lemme add both so they cant miss

Reply:
oop I misunderstood you originally! I got it now I think